### PR TITLE
[opt](function)Optimize the Percentile function.

### PR DESCRIPTION
**Pull request by Mryange**
## Proposed changes
Support batch functions and floating-point parameters.

### CODE REVIEW

**Comment by Mryange** [comment]
run buildall

**Comment by Mryange** [comment]
run buildall

File: fe/fe-core/src/main/java/org/apache/doris/nereids/trees/expressions/functions/agg/Percentile.java [hunk 1]
```diff
@@ -42,6 +43,8 @@ public class Percentile extends NullableAggregateFunction
         implements BinaryExpression, ExplicitlyCastableSignature {
 
     public static final List<FunctionSignature> SIGNATURES = ImmutableList.of(
+            FunctionSignature.ret(DoubleType.INSTANCE).args(DoubleType.INSTANCE, DoubleType.INSTANCE),
+            FunctionSignature.ret(DoubleType.INSTANCE).args(FloatType.INSTANCE, DoubleType.INSTANCE),
```
Comment (superdiaodiao) [inline fe/fe-core/src/main/java/org/apache/doris/nereids/trees/expressions/functions/agg/Percentile.java:47]: Better add tests for these two cases.

**Comment by Mryange** [comment]
run cloud_p0

**Comment by Mryange** [comment]
run buildall

**Comment by Mryange** [comment]
run buildall

**Review by HappenLee — APPROVED** [review]
LGTM

**Comment by Mryange** [comment]
run p0

**Comment by Mryange** [comment]
run buildall

**Review by zhangstar333 — APPROVED** [review]

**Pull request merged**